client/helpers/scenes.ts

```diff
@@ -300,54 +324,118 @@ export async function createSceneAndSceneItemsEXPERIMENTAL({
   polling = { intervalMs: PollIntervalMs, maxAttempts: MaxAttempts },
   returnQueued = false,
   verbose,
-}: CreateSceneAndSceneItemsReq): Promise<CreateSceneAndSceneItemsResEXPERIMENTAL> {
+}: CreateSceneAndSceneItemsReqEXPERIMENTAL): Promise<CreateSceneAndSceneItemsResEXPERIMENTAL> {
+  const startTime = hrtime.bigint();
+  if (verbose) onMsg(`Creating scene...`);
   const scene = (
     await client.scenes.createScene({ createSceneRequest: createSceneReq() })
   ).data;
   const sceneId = scene.data.id;
-  const createRes = await createSceneItemsEXPERIMENTAL({
-    client,
-    createSceneItemReqs,
-    failFast,
-    onProgress,
-    parallelism,
-    sceneId,
-  });
-  const { a: queuedOps, b: errors } = partition(createRes.queuedBatchOps, (i) =>
-    isQueuedJob(i.res)
-  );
 
-  const queued = returnQueued ? createRes.queuedBatchOps : [];
-  // Nothing succeeded, return early as something is likely wrong
-  if (queuedOps.length === 0 || errors.length === createRes.chunks) {
-    return { errors, queued, scene, sceneItemErrors: [] };
-  }
-
-  if (verbose) onMsg(`Polling for completed scene-item batches...`);
+  if (verbose) onMsg(`Creating scene items...`);
+  let itemCount = 0;
+  let batchQueuedOps: QueuedBatchOps[] = [];
+  let batchErrors: QueuedBatchOps[] = [];
```

Comment:
Would be ideal here if this implementation were immutable.

Reply:
True, this could be done as a const array. But I'm going to leave it for now.